Src/Support/Google.Apis.Core/GoogleApiException.cs

```diff
@@ -26,6 +26,7 @@ namespace Google
     public class GoogleApiException : Exception
     {
         private readonly string serviceName;
```

Comment:
Let's remove this and make ServiceName a read-only autoprop. That remove the unconventional field name and makes the code feel more modern :)